site/load-chart.js

```diff
@@ -44,11 +44,7 @@ function toolTip (date, wpt_sha, servo_version, score, engine) {
 }
 
 function setupChart () {
-    const endOfYear = new Date(2024, 11, 31)
     let maxDate = new Date()
```

Comment:
I guess this can be changed to a `const` now as we don't modify it after assignment. Worth running `npm run lint` so it doesn't fail in the CI.

Reply:
Done. Thanks!